From 62c88b1ad43dfec123818e8de7a9290397d6be65 Mon Sep 17 00:00:00 2001
From: Debarshi Ray <debarshir@gnome.org>
Date: Mon, 30 Jan 2017 14:37:16 +0100
Subject: configure: Require libsecret

In the following patch, we will need libsecret to store the user
passwords during an express installation.

https://bugzilla.gnome.org/show_bug.cgi?id=777788
---
 configure.ac    | 1 +
 src/Makefile.am | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 6dfa38c..3ca3ebc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -71,6 +71,7 @@ PKG_CHECK_MODULES(BOXES, [
   gobject-introspection-1.0 >= $GOBJECT_INTROSPECTION_MIN_VERSION
   gtk+-3.0 >= $GTK_MIN_VERSION
   gtk-vnc-2.0 >= $GTK_VNC_MIN_VERSION
+  libsecret-1
   libvirt-gobject-1.0 >= $LIBVIRT_GLIB_MIN_VERSION
   libvirt-gconfig-1.0 >= $LIBVIRT_GCONFIG_MIN_VERSION
   libxml-2.0 >= $LIBXML2_MIN_VERSION
diff --git a/src/Makefile.am b/src/Makefile.am
index 4f3e43e..bd735f2 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -32,6 +32,7 @@ AM_VALAFLAGS =						\
 	--pkg gtk-vnc-2.0				\
 	--pkg gudev-1.0					\
 	--pkg libarchive				\
+	--pkg libsecret-1				\
 	--pkg libsoup-2.4				\
 	--pkg libosinfo-1.0				\
 	--pkg libvirt-gconfig-1.0			\
-- 
cgit v0.12

